packages/wonder-blocks-modal/src/components/modal-header.tsx

```diff
@@ -124,6 +124,7 @@ export default function ModalHeader(props: Props) {
                 <View style={styles.breadcrumbs}>{breadcrumbs}</View>
             )}
             <HeadingMedium
+                tag="h2"
```

Comment:
**suggestion:** Now that this change is in place, I think it would be helpful to add a new unit test in `modal-header.test.tsx` to verify that the title renders and `h2`. This would be helpful for us in the future to prevent regressions.

You can see the existing tests in that file for reference. Something like this could work:

```
...
expect(screen.getByRole("heading", {level: 2, text: "Title"}).toBeInTheDocument();
```

Reply:
Unit test added!